lib/cosmos/gui/text/ruby_editor.rb

```diff
@@ -141,12 +147,8 @@ def initialize(editor)
         self
       end
 
-      def sizeHint()
-        return Qt::Size(@codeEditor.lineNumberAreaWidth(), 0)
```

Comment:
This isn't even syntactically correct (no new) so obviously we've never called it.